packages/vscode-extension/package.nls.json

```diff
@@ -101,6 +101,8 @@
   "teamstoolkit.commands.selectTutorials.title": "View How-to Guides",
   "teamstoolkit.commands.switchTenant.m365.title": "Switch between your available tenants for Microsoft 365 account",
   "teamstoolkit.commands.switchTenant.azure.title": "Switch between your available tenants for Azure account",
+  "teamstoolkit.commands.switchTenant.progressbar.title": "Switch tenant",
+  "teamstoolkit.commands.switchTenant.progressbar.detail": "Teams Toolkit is switching from your current tenant to the newly selected tenant.",
```

Comment:
Proposed content: Teams Toolkit is now switching to the newly selected tenant.

Reply:
Updated